.github/workflows/integration_tests.yml

```
@@ -134,6 +132,7 @@ jobs:
                   echo ":x: Test failed (failed to compile): $filename" >> $GITHUB_STEP_SUMMARY
                   test_err_sum=$((test_err_sum+1))
                 fi
+	      ;;
```

Looks like some tabs snuck into a few spots.
```suggestion
              ;;
```